snyk/cli
# feat: switch to zerolog

#### What does this PR do?
In this PR we update the application framework to make use of it's support of zerolog and start using zerolog in the CLI.

#### Where should the reviewer start?
main.go

#### How should this be manually tested?
run the cli with -d and without, in general the appearance should not change for now.
